gt/main.c

```diff
@@ -1433,11 +1429,13 @@ gt_stage2(void *arg)
 }
 
 int
-run_gt(struct net_config *net_conf, struct gt_config *gt_conf)
+run_gt(const char *policy_dir_name, const char *gt_config_file_name,
```

Comment:
If you take the suggestion to rename the variables above, then these should be renamed too.